templates/common/.devcontainer/devcontainer.json/java/devcontainer.json

```diff
@@ -3,16 +3,13 @@
     "build": {
         "dockerfile": "Dockerfile",
         "args": {
-            "VARIANT": "bullseye"
+            "IMAGE": "java:17"
```

Comment:
@pamelafox , looks like `Maven` is missing

We are getting:

```
Deploying apps for azd-template-test-todo-java-mongo-2644080-1...


ERROR: required external tools are missing:
 - error checking for external tool Maven: maven could not be found. Install either Maven or Maven Wrapper by visiting https://maven.apache.org/ or https://maven.apache.org/wrapper/

### ERROR: Dev container exec failed (exit code: 1)

```

Any idea how to fix it?  or should we revert the change for Java for now?